css-borders-4/Overview.bs

```diff
@@ -336,29 +336,178 @@ The 'border-radius' shorthand</h4>
+Since stroking a superellipse accurately may be computationally intensive, user agents may approximate the path using bezier curves,
+as well as account for sharp edges and overlaps.
+
+Note that `border-radius` already has a <a href="https://drafts.csswg.org/css-backgrounds/#corner-overlap">corner overlap restriction</a>,
+and it's unclear whether 'corner-shape' would require additional restrictions.
+
+Issue <a href="https://github.com/w3c/csswg-drafts/issues/11610">#11610</a>: check if we need additional rendering restrictions.
+
+<h4 id=corner-shape-value>
+'corner-shape' values</h4>
+
+	<pre class=prod>
+		<dfn><<corner-shape-value>></dfn> = ''round'' | ''scoop'' | ''bevel'' | ''notch'' | ''straight'' | ''squircle'' | superellipse(<<number [0,&infin;]>>)
+	</pre>
+
+	<dl dfn-type="value" dfn-for="corner-shape-value">
+		<dt><dfn>''round''</dfn>
+		<dd>
+			Border radii define a convex elliptical curve at the corner. Equivalent to <css>superellipse(2)</css>.
+
+			Note: this is the initial value of 'corner-shape' properties, as elements with 'border-radius' would be rounded.
+
+		<dt><dfn>''scoop''</dfn>
+		<dd>Border radii define a concave elliptical curve at the corner. Equivalent to <css>superellipse(0.5)</css>.
+		<dt><dfn>''bevel''</dfn>
+		<dd>Border radii define a diagonal slice at the corner. Equivalent to <css>superellipse(1)</css>.
+		<dt><dfn>''notch''</dfn>
+		<dd>Border radii define a concave 90deg angle at the corner. Equivalent to <css>superellipse(0)</css>.
+		<dt><dfn>''straight''</dfn>
+		<dd>Border radii define a convex 90deg angle at the corner.
+			This would have the same visual effect as a 'border-radius' of 0. This is different from having a 'border-radius' of 0 when animating.
+			Equivalent to <css>superellipse(infinity)</css>.
+		<dt><dfn>''squircle''</dfn>
+		<dd>Border radii define a convex curve between an ellipse and an convex angle, equivalent to <css>superellipse(4)</css>.
+	</dl>
+
+	Issue <a href="https://github.com/w3c/csswg-drafts/issues/11607">#11607</a>: resolve on ''straight'' vs <css>none</css>.
+
+	The <dfn>superellipse()</dfn> function describes the curvature of the corner.
+	It accept a <<number>> <dfn>superellipse exponent</dfn>, which defines the curvature of the corner, or the exponent of the formula.
+	The [=superellipse exponent=] accepts values between 0 (a straight concave angle) and <css>infinity</css> (a straight convex angle),
+	with the values in between representing the curvatures in between.
+
+	The ''superellipse()'' formula can be described in cartesian coordinates, as follows,
+	where <code>a</code> is the horizontal ''border-radius''
+	<code>b</code> is the vertical ''border-radius'', and <code>k</code> is the [=superellipse exponent=]:
+
+	<pre>
+		|x/a|^k + |y/b|^k = 1
+	</pre>
+
+
+<h4 id=corner-shape-properties>
+Corner Shaping: the 'corner-*-*-shape' properties</h4>
```

Comment:
While we *usually* do longhands first, followed by the shorthand, we tend to use the opposite order when the shorthand is going to be used the most often and the longhands are just it exploded to every possible side/corner. Mind swapping the order of these? They can all be grouped under one heading, maybe "Corner Shaping: the 'corner-shape' and 'corner-*-shape' properties".

Reply:
Sure!

Reply:
Done